### ISSUE TITLE
Build a step by step readme that allows any new person on the team to run atleast 2 Pyrsia nodes with a client and demonstrate all currently working functionality

### ISSUE THREAD

**Issue by betarelease** [pyrsia/pyrsia#294]
I tried to follow the steps in the readme and ran into a couple of issues where I had to discover the correct command/inputs before I could run the demo successfully.

Here is a list of items that need to be addressed:

1. Write down step by step instructions for a new developer to get the pyrsia nodes(atleast 2 of them) running and interacting with the CLI
2. Write down clear instructions on how Synapse needs to be brought up and how a magneto URL can be submitted to see streaming large files in action
3. Include success criteria by adding examples of expected output for each step above

Ensure that the resulting README.md addresses the following questions

- Going to install it using the readme.
- What platform is supported?
- 2 options of building - which one is better when?
- Since I am not on Ubuntu, I'm going with Docker.
- dbuild 2/2 takes a REALLY long time. Maybe worth mentioning.
- For registry-mirrors to take effect, I have to restart the docker engine. It kills the existing docker run. Should I do the registry-mirrors before I build? 
- I restarted the container after restart of the engine (this should be a step, if we don’t move the config pre-build)
- It’s better to use a smaller image to test the setup. Busybox?
- How do I know the image pull was using Pyrsia? Is it something I’ll do with the CLI? Worth mentioning.
- CLI build failed (xcrun is missing).
- Fixed by running xcode-select --install
- Compilation produced one warning: 
- warning: path statement with no effect
-   --> pyrsia_cli/src/main.rs:29:5
-    |
- 29 |     model::package_version::LicenseTextMimeType::Text;
-    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-    |
-    = note: `#[warn(path_statements)]` on by default
- If we change the sample image, we need to change the output of node -s to match

Please split this ticket into multiple smaller tickets so that this can be built in parallel

#297

**Comment by jbaruch** [comment]
I've added a couple of issues about the main points in the list in #290 and #288.

**Comment by jbaruch** [comment]
I can edit the readme to address those if y'all ok with that.

**Comment by MitaliBo** [comment]
@jbaruch we welcome all feedback , please go ahead and create PR

**Comment by johanvos** [comment]
I created #308  and PR #309 as a fix for a smaller issue that developers can run into.
The main bottleneck I see with this issue is that it doesn't mention how to demonstrate the P2P yet, or do we tackle this in another issue?

**Comment by prince-chrismc** [comment]
Having gone through #370 and all the linked issues are merged... is this not already done?
It's not clear whats missing 😖

**Comment by betarelease** [comment]
Link PRs from installation documentation and verify if this particular issue is addressed @betarelease
